lib/insights_cloud/async/cloud_connector_announce_task.rb

```diff
@@ -0,0 +1,47 @@
+module InsightsCloud
+  module Async
+    class CloudConnectorAnnounceTask < ::Actions::EntryAction
+      def self.subscribe
+        Actions::RemoteExecution::RunHostsJob
+      end
+
+      def self.connector_feature_id
+        @connector_feature_id ||= RemoteExecutionFeature.feature!(ForemanRhCloud::CloudConnector::CLOUD_CONNECTOR_FEATURE).id
+      end
+
+      def plan(job_invocation)
+        return unless connector_playbook_job?(job_invocation)
+
+        plan_self
+      end
+
+      def run(event = nil)
```

Comment:
To be honest I'm not sure how subscribed actions behave. Wouldn't this run in parallel to `RunHostsJob` that triggered this one?